kong/clustering/tls.lua

```diff
@@ -226,7 +228,7 @@ function tls.validate_client_cert(kong_config, cp_cert, dp_cert_pem)
     return nil, err
   end
 
-  return true
+  return cert
```

Comment:
It's better to keep the number of return values consistent.
```suggestion
  return cert, nil
```

Reply:
doen